wordpress-mobile/WordPress-iOS
# [Gravatar] Fix gravatar name conflict

Fixes https://github.com/gravatar/Gravatar-SDK-iOS/issues/23

Renaming the `Gravatar` struct to `GravatarURL` to make integration tests of the `Gravatar` SDK easier.

WPUP side: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/137

### To test:
Checkout `gravatar/etoledom/fix-gravatar-name-conflict` on [WPUI](https://github.com/wordpress-mobile/WordPressUI-iOS)
Project should Build and Run.
Test manually to change your avatar nuder `My Profile` section
